src/language.ts

```diff
@@ -3,94 +3,129 @@ import stringDirection from 'string-direction';
 
 import type { Tag, Subtag } from 'language-tags';
 
-class LanguageObj {
+class Language {
+    /**
+     * Included for backwards compatibility, prefer:
```

Comment:
Why go for backwards compatibility here? I thought these were all clearly implementation details?

And doesn't `LanguageObj` -> `Language` break compatibility?